src/goto-harness/goto_harness_generator.h

```diff
@@ -16,6 +16,27 @@ Author: Diffblue Ltd.
 
 class goto_modelt;
 
+class harness_options_parsert
```

Comment:
Hmm, so this really is using a class as a namespace, just because our coding guidelines say so?

Reply:
It was not the intention, but I guess that is the only reason.

Reply:
Then maybe we should actually use a namespace?

Reply:
Here: https://github.com/diffblue/cbmc/pull/4482/commits/791c44292d31a344d26798538e00b91117ed664f. I'm just not sure about the naming convention: should a namespace end with `t` or not?

Reply:
Also the linter doesn't like it very much.